packages/office-ui-fabric-react/src/components/Keytip/Keytip.styles.ts

```diff
@@ -0,0 +1,77 @@
+import { IKeytipStyleProps, IKeytipStyles } from './Keytip.types';
+import { ICalloutContentStyleProps, ICalloutContentStyles } from '../../Callout';
+import { IStyleFunction, IPoint } from '../../Utilities';
+
+export const getStyles = (props: IKeytipStyleProps): IKeytipStyles => {
+  const { theme, disabled, visible } = props;
+  return {
+    container: [
+      {
+        backgroundColor: theme.palette.neutralDark
+      },
+      disabled && {
+        opacity: 0.5,
+      },
+      !visible && {
+        visibility: 'hidden',
+        opacity: 0
+      }
+    ],
+    root: [{
+      textAlign: 'center',
+      paddingLeft: 3,
+      paddingRight: 3,
+      backgroundColor: theme.palette.neutralDark,
+      color: theme.palette.neutralLight,
+      minWidth: 11,
+      lineHeight: 17,
+      height: 17,
+      display: 'inline-block'
+    },
+    disabled && {
+      color: '#b1b1b1'
```

Comment:
use a color variable

Reply:
What would be the correct pattern here? Should I add a color to the DefaultPalette or should I hardcode a semantic color?

Reply:
https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/styling/src/interfaces/ISemanticColors.ts